src/StdUtils.sol

```diff
@@ -133,6 +135,21 @@ abstract contract StdUtils {
         return computeCreate2Address(salt, initCodeHash, CREATE2_FACTORY);
     }
 
+    /// @dev returns a initilized mock ERC20 contract
+    function deployERC20Mock(string memory name, string memory symbol, uint8 decimals)
+        internal
+        returns (ERC20Mock mock)
+    {
+        mock = new ERC20Mock();
```

Comment:
Let's rename everything to put the "Mock" part as a prefix, e.g. `MockERC20` and `MockERC721`—this is the naming convention solmate uses, and it's my preference as well. For example:
```suggestion
    function deployMockERC20(string memory name, string memory symbol, uint8 decimals)
        internal
        returns (MockERC20 mock)
    {
        mock = new MockERC20();
```